sweep/sweeper.go

```diff
@@ -323,11 +321,26 @@ func (s *UtxoSweeper) Start() error {
 	// not change from here on.
 	s.relayFeeRate = s.cfg.FeeEstimator.RelayFeePerKW()
 
-	// Register for block epochs to retry sweeping every block.
-	bestHash, bestHeight, err := s.cfg.ChainIO.GetBestBlock()
+	// We need to get the current best block so we can register for block
+	// epochs and retry sweeping every block. We should get a notification
+	// immediately if we don't provide any epoch.
```

Comment:
The rationale is that we don't want to use `GetBestBlock` any more because it is not exposed through any RPC at the moment. So by using the chain notifier to get the current best block we have everything provided through RPCs that the sweeper needs to work outside of lnd.
I updated the commit message to make this clearer.